crux-http-server/src/crux/http_server/entity_ref.cljc

```diff
@@ -26,11 +26,6 @@
       (.write w "#crux.http/entity-ref ")
       (print-method (:eid ref) w)))
 
-#? (:clj
```

Comment:
Question: Is print-dup EntityRef now completely unused?

Reply:
I learned that we can't use `print-dup` for EDN, because it tries to preserve the concrete types (e.g.) of maps - `{:a 1, :b 2}` gets printed as `#=(clojure.lang.PersistentArrayMap/create ...)`, which isn't valid EDN.

Will double-check that we don't use it, though